src/images/ic_airplay_black_24px.svg

```diff
@@ -0,0 +1,15 @@
+<svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <defs>
+        <path d="M0 0h24v24H0V0z" id="a"/>
+    </defs>
+    <defs>
+        <path d="M0 0h24v24H0V0z" id="c"/>
+    </defs>
+    <clipPath id="b">
+        <use overflow="visible" xlink:href="#a"/>
+    </clipPath>
+    <clipPath clip-path="url(#b)" id="d">
+        <use overflow="visible" xlink:href="#c"/>
+    </clipPath>
+    <path clip-path="url(#d)" d="M6 22h12l-6-6zM21 3H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h4v-2H3V5h18v12h-4v2h4c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"/>
+</svg>
```

Comment:
Can you go ahead and fix the missing trailing newline on the SVGs that are missing it? Seems that one has it and two don't.